test/lib/nvme/overhead/overhead.c

```diff
@@ -39,9 +39,6 @@
 #include <string.h>
 #include <unistd.h>
 
-#include <rte_config.h>
```

Comment:
I cherry-picked this commit as decea46f594a0b487a0830fbbcce5f32b3b33b27, thanks!